src/components/ErrorBox/ErrorBox.tsx

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from "react";
+import styles from '@/components/ErrorBox/ErrorBox.module.css';
+
+export interface ErrArrType {
```

Comment:
```suggestion
export interface ErrorBoxProps {
```

There's this convention where Component props are named after the component with the Props suffix.

Reply:
Done.